.circleci/config.yml

```diff
@@ -553,6 +578,16 @@ workflows:
             - mysql-integration-test
             - postgres-integration-test
           filters: *filter-only-release
+      - build-oss-msi:
```

Comment:
We need a dependency from `deploy-release` to `build-oss-msi` so that we can be sure that the msi is built before deploying.